.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Release .deb files'
+on: release
+
+jobs:
+  release-bionic:
```

Comment:
You can avoid these repeated release-Xs with strategy e.g.
```
strategy:
  matrix:
    os:
    - name: bionic
       deb: libnginx-mod-http-sxg-filter_1.14.0-0ubuntu1.6_amd64.deb
    - name: eoan
...
```

Reply:
done.